stable/memcached/templates/svc.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
+  clusterIP: None
```

Comment:
Yes statefulset requires Headlessservices and [this ](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services)is the way to create headless services